core/vm/evm.go

```diff
@@ -638,6 +638,9 @@ func (evm *EVM) CreateETX(toAddr common.Address, fromAddr common.Address, gas ui
 	}
 	conversion := false
 	if toAddr.IsInQiLedgerScope() && common.IsInChainScope(toAddr.Bytes(), evm.chainConfig.Location) { // Quai->Qi Conversion
+		if evm.Context.BlockNumber.Cmp(new(big.Int).SetUint64(params.ControllerKickInBlock)) < 0 {
```

Comment:
is this enough to do in the create etx? no need to do this in the `opEtx` ?

Reply:
I think you're thinking of opConvert. opETX should already not work without any extra logic. But I fixed it